share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -321,16 +342,25 @@ function(ExternalZephyrProject_Cmake)
          "${ZCMAKE_APPLICATION}_BOARD:STRING=${${ZCMAKE_APPLICATION}_BOARD}\n"
     )
   endif()
+  list(APPEND sysbuild_cache_strings
+       "${ZCMAKE_APPLICATION}_KCONFIG_BINARY_DIR:PATH=${${ZCMAKE_APPLICATION}_KCONFIG_BINARY_DIR}\n"
+  )
 
   file(WRITE ${${ZCMAKE_APPLICATION}_CACHE_FILE}.tmp ${sysbuild_cache_strings})
   zephyr_file_copy(${${ZCMAKE_APPLICATION}_CACHE_FILE}.tmp
                    ${${ZCMAKE_APPLICATION}_CACHE_FILE} ONLY_IF_DIFFERENT
   )
 
+  set(dotconfigsysbuild ${${ZCMAKE_APPLICATION}_KCONFIG_BINARY_DIR}/.config.sysbuild)
+  get_target_property(config_content ${ZCMAKE_APPLICATION} CONFIG)
+  string(CONFIGURE "${config_content}" config_content)
+  file(WRITE ${dotconfigsysbuild} ${config_content})
```

Comment:
Writes `config_content-NOTFOUND` if empty.

Reply:
nice catch, guess it happened as side-effect on the board revision issue.

Will ensure the `CONFIG` setting is always initialized with a value.

Reply:
Fixed.